src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type ControllerResponse = {
+  isSuccess: number;
+  statusCode: number;
+  message: string;
+  data?: any;
+  error?: any;
+};
```

Comment:
PullProofAI

Consider using interfaces instead of type aliases for defining shapes of objects. Interfaces are more suitable for defining object shapes in TypeScript.

```diff
- export type ControllerResponse = {
+ export interface ControllerResponse {
```



<!-- This is an auto-generated comment by PullProofAI -->